backend/config/gene-info-api.yml

```diff
@@ -0,0 +1,72 @@
+openapi: 3.0.3
+info:
+  version: "1.0.0"
+  title: Chan Zuckerburg Initiative cellxgene Gene Info API
+  description: >-
+    This API is for internal use to access information from NCBI's E-Utilities API about a given gene.
+servers:
+  - description: Local
+    url: /
+  - description: Production environment
+    url: https://api.cellxgene.cziscience.com/
+  - description: Development environment
+    url: https://api.dev.single-cell.czi.technology/
+  - description: Staging environment
+    url: https://api.staging.single-cell.czi.technology/
+
+paths:
+  /v1/gene_info:
+    get:
+      summary: Given a gene ensembl ID, returns that gene's summary information, full name, synonyms, and NCBI URL. 
+      description: >-
+      tags:
+        - gene_info
+      operationId: backend.gene_info.api.v1.gene_info
+      parameters: 
+        - $ref: '#/components/parameters/GeneID'
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GeneInfo'
+        "404":
+          $ref: "#/components/responses/404"
+
+components:
+  schemas:
+    problem:
```

Comment:
We should use the same error format across the APIs https://github.com/chanzuckerberg/single-cell-data-portal/blob/ff7ebfb7218c4f26176fde905613bfa0c73d4712/backend/config/corpora-api.yml#L1244-L1253